index.js

```diff
@@ -5,11 +5,15 @@ globals.forEach(function(g) {
   if (g in global) globalValues[g] = global[g];
 });
 
-require(process.env['LATER_COV'] ? "./later-cov" : "./later");
+if (process.env['LATER_COV']) {
```

Comment:
you should also check if process is defined before trying to access env. webpack will throw an error here. I did the same fix in #208